WooCommerce/Classes/ViewRelated/Custom Fields/CustomFieldEditorView.swift

```diff
@@ -22,13 +24,20 @@ struct CustomFieldEditorView: View {
     ///  - value: The value for the custom field
     ///  - isReadOnlyValue: Whether the value is read-only or not. To be used if the value is not string but JSON.
     ///  - onSave: Closure to handle save action
-    init(key: String, value: String, isReadOnlyValue: Bool = false, onSave: @escaping (String, String) -> Void) {
+    init(key: String,
+         value: String,
+         isReadOnlyValue: Bool = false,
+         isCreatingNewField: Bool = false,
+         onSave: @escaping (String, String) -> Void,
+         onDelete: @escaping () -> Void = {}) {
```

Comment:
> Also, is there any reason why onDelete has a default value {} while onSave doesn't? Should we keep them consistent?

This is because `onDelete` is optional while `onSave` is not, `onDelete` can be omitted when we are creating a new field, but given I'm making it nullable later, I think it would be more clear now.